src/darwin/Framework/Matter.xcodeproj/project.pbxproj

```diff
@@ -2032,21 +2032,22 @@
 					"CHIP_ADDRESS_RESOLVE_IMPL_INCLUDE_HEADER=<lib/address_resolve/AddressResolve_DefaultImpl.h>",
 					"$(inherited)",
 				);
+				GCC_SYMBOLS_PRIVATE_EXTERN = YES;
```

Comment:
Hmm....  According to https://developer.apple.com/library/archive/documentation/DeveloperTools/Reference/XcodeBuildSettingRef/1-Build_Setting_Reference/build_setting_ref.html, "YES" is the default... but ok.

Reply:
I think that's an outdated document. According to how Xcode resolves this, the default is NO (and that's the behavior in practice.)